core/arch/arm/include/sm/optee_smc.h

```diff
@@ -290,6 +290,12 @@
 #define OPTEE_SMC_SEC_CAP_HAVE_RESERVED_SHM	(1 << 0)
 /* Secure world can communicate via previously unregistered shared memory */
 #define OPTEE_SMC_SEC_CAP_UNREGISTERED_SHM	(1 << 1)
+/*
```

Comment:
Based on the mailing list discussion, we don't need both OPTEE_SMC_SEC_CAP_UNREGISTERED_SHM and OPTEE_SMC_SEC_CAP_DYNAMIC_SHM, right?

Jens had proposed changing the existing "UNREGISTERED_SHM" capability to this:

/*
  * Secure world supports shared memory objects to represent
  * physically non-contiguous memory and the commands
  * "register/unregister shared memory object" for more efficient
  * communication via physically non-contiguous memory.
  */
#define OPTEE_SMC_SEC_CAP_NONCONTIG_SHM  (1 << 1)